templates/cassiopeia/html/layouts/chromes/cardGrey.php

```diff
@@ -21,34 +21,22 @@
 }
 
 $moduleTag              = $params->get('module_tag', 'div');
-$moduleAttribs          = [];
+$modulePos 		= $module->position;
```

Comment:
```suggestion
$modulePos              = $module->position;
```
PHPCS: Here we use spaces to indent to alignt the "=" if several assignments in consecutive lines.

Reply:
This one here, too, please.